src/content/docs/service-level-management/intro-slm.mdx

```diff
@@ -10,15 +10,22 @@ metaDescription: How to create and consume SLIs and SLOs with service level mana
 
 import slmServiceLevelsOverview from 'images/slm_screenshot-full_service-levels-overview.png'
 
-With New Relic you can define and consume service level indicators and service level objectives for your applications.
+Service levels were popularized by Google and became a best practice in the industry. Service levels complement existing monitoring by adding a perspective more focused on the business and the user experience.
+
+With New Relic, you can define and consume service level indicators (SLIs) and service level objectives (SLOs) for your applications. There are several relevant reasons to configure and follow your New Relic service levels:
+
+- Easy creation: facilitates creation of service levels with different modes of complexity, from one-click to advanced modes fully customizable for advanced users. 
+- Integrated with New Relic experiences: visualize and work with services levels in Navigator, Workloads, mini overview and through most of the New Relic observability tools.
```

Comment:
```suggestion
- Integrated with New Relic experiences: visualize and work with services levels in New Relic Navigator, workloads, mini overview and through most of our observability tools.
```

Reply:
I'm unsure. Should I change this to just say "Navigator and  Workloads."